### PR TITLE
#171098744 Change user booking response

### PR DESCRIPTION
**Pull request by victkarangwa**
#### What does this PR do?
This PR changes user booking request and modifies email template for user verification
#### Description of Task to be completed?
- Include accommodation information
- Include room information
- Remove user trip from the booking request
- Modify verification email template
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- git checkout ch-change-booking-response-171098744
- `npm install`
- Make sure your `.env` file is updated.
- Drop existing migration `npm run dropTables`
- Update migration `npm run db-migrate`
- Populate the seed data `npm run db-seed-dev`
- `npm run dev-start` check API in development 
>Create booking `{{url}}/api/v1/bookings`.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171098744](https://www.pivotaltracker.com/story/show/171098744)
##### Response after user booking
Screen Shot 2020-02-10 at 15 39 58

##### User verification email
Screen Shot 2020-02-20 at 09 55 59

#### Questions:
N/A

### CODE REVIEW

**Review by william000000 — APPROVED** [review]
Good Job @victorkarangwa4

**Review by NiyongaboEric — APPROVED** [review]
Good initiative @victorkarangwa4 !

**Review by NiyoEric — APPROVED** [review]
Awesome job!

**Review by hezronkimutai — APPROVED** [review]
Nice work @victorkarangwa4

File: src/helpers/UserHelper.js [hunk 1]
```diff
@@ -58,6 +58,15 @@ class UserHelper {
...
       });
+      const { id } = socialUser[0].dataValues;
+      const k = await userProfile.findOrCreate({
+        where: { userId: id },
+        defaults: {
+          userId: id,
+        }
+      });
+      console.log('====>', k);
+
```
Comment (william000000) [inline src/helpers/UserHelper.js]: Remove this console

File: src/controllers/BookingController.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ class BookingController {
       await NotificationService.newBookingNotification(req);
       return Response.successMessage(req, res, 'You have booked an accommodation facility successfully', result, 201);
     } catch (error) {
+      console.log(error);
```
Comment (nziokaivy) [inline src/controllers/BookingController.js]: Please remove this console log

File: src/helpers/UserHelper.js [hunk 3]
```diff
@@ -58,6 +58,15 @@ class UserHelper {
...
         }
       });
+      const { id } = socialUser[0].dataValues;
+      const k = await userProfile.findOrCreate({
+        where: { userId: id },
+        defaults: {
+          userId: id,
+        }
+      });
+      console.log('====>', k);
```
Comment (nziokaivy) [inline src/helpers/UserHelper.js]: remove this console.log as well

**Review by nziokaivy — CHANGES_REQUESTED** [review]
Great work @victorkarangwa4! Just remove those console logs and you are good to go!

**Pull request merged**